native/storybook/storybook.js

```diff
@@ -5,11 +5,13 @@ import { getStorybookUI, configure } from '@storybook/react-native';
 configure(() => {
   require('../packages/react-native-bpk-component-text-input/stories');
   require('../packages/react-native-bpk-component-text/stories');
+  require('../packages/react-native-bpk-component-button/stories');
 }, module);
 /* eslint-enable global-require */
 
+// This assumes that storybook is running on the same host as your RN packager,
+// to set manually use, e.g. host: 'localhost' option
```

Comment:
We did a bad rebase here - we overwrote changes that we shouldn't have.
My fault as much as anything!